browser-test/src/support/cookies.ts

```diff
@@ -0,0 +1,23 @@
+  'https://staging-aws.civiform.dev':
+    'eyJhbGciOiJIUzI1NiJ9.eyJkYXRhIjp7InNlc3Npb25JZCI6ImZmYmI3YzYwLTUyYmYtNGZhYS04Y2I0LTI1YjU5ZmRlZmIxMSIsInBhYzRqIjoiaTdqZ01TL2VLS2JFdVBLUzREVU5DcWZzOVBTVFAxMTNqdUFBZW5IcksvcFE4VXhIZGYyQ0dmbEVYVnVlUUowZmJ1bkdtVGsreFg4ZEYyOVFlbThCTWxBcDlKWU9KZUpvWFQ4WHQ2YS9lV0RXa3dZbGFrWWpGVklKSmtaYW05RDVneGZiMjF3em85SHNSUnBYOXQ0YkNZdVIrRmtucjdRWjFHTkw1NTVPY0pMaVR0OG05TXVQQk5BcVhoMzRXSG55QXliQUFsRGxTWGI1bnE1Uk9LWmxoNkc5ZDgrSW1vMlBzSTlZOXdKUDkxYTJUWmxZTjdpME5taGVtdFBld1ZzVUxMQTBXZ2lTK1oxelplRTZlQ2FPRTk3NXBPVlo0dUh1RkFvL3lGRFZoM1VIRDUzYVpmYi9vUm1sKy80cVVMYVcyd3pLdGhzS012bHNmWGRjYXZGM0hpSVYvS0J3KzdJWkdvNUJpK2xPeDIvUDVFTWk3bERGeUN2QW14VHU4dHVLTy9HYmdwOUgvSXJmUnNhNTNkUUJ1V0hhM1Y1NHJOZE5BYVA4N0dXWTJoRVJld2hsUDBFc29ZM1p3QzZXN1c5S1gwUnBLNEs2VncxNllXN2JpT1Z3N29MOHdRMXlQaDRDN1MydjFtcTlvZUFBZWNZNXJVd1FiVTIvb3ZiVTJ4bTU2QlVrd2o0Tzd4emVQOFpydUdkNkdDRk1jbXlwVU1yWVQwbnYyK003OWJzM2t5L1NHdHlvK3hmS09KR09taXIzaHVSTUc5cUpTaHZkcmVCemVKbFJwbFVNd0kxTXBSVWdmMkgrelFuSmNwallKS2pUZnY0MURsd0RBYnNHRUVFTmQ3WVVVMGRMVzJRK0dEWTFWYjhlM1VpVXdpVjJ1YjRaWkhubTFwRytnUmViemxsbFVuRkFQa2hFL0NXZVRUWVhJQkIxS3E1bmxFTVBZcDIxeDFBRkIwMzJhcG1LVDRaWkQ1djRncm45YUE9PSIsImNzcmZUb2tlbiI6IjE2NTQ5MTMwOTA0MzZiNzA2M2MwZDU1ZDNkNzM1MTViNGFlN2IzYjAtMTcwMjMwMzUwNjE4MS00ZTA0N2VhMzYwMzg5OGYzY2MwMzQ5NzEifSwibmJmIjoxNzAyMzA5MzgxLCJpYXQiOjE3MDIzMDkzODF9.zVhM28Q6O3mRXOTrctH07_pBTI7doBwCoE2Kj-7TqY0',
+  'https://civiformstage.seattle.gov':
+    'eyJhbGciOiJIUzI1NiJ9.eyJkYXRhIjp7InNlc3Npb25JZCI6IjU1ZTRhNjdlLTgzYTUtNDY2Mi1hMmM5LTBlZGE3ZGQ0ZDI4OCIsInBhYzRqIjoic1pNa2h4ZENpVGUybUsxNmFhMFZhVHdaSHNwUyt5S1hWZkltS2syM0lpSEpGaE5jWjQvY2xYb0xEK3pFVDJSWEpsczc0dUh4bjl6T1FqUGtDdm9nalJoUXpmR0haTUM1R2xMc1ExRFBHS3E5U2NwMW5NUjc0MFhDcElnSUlxK2xjL0NyMlhkUlhpb1phVTlSdXBIVFlLZEtPSmszVGpTYXMzelZPNjZBQlJYQ3dSSDVrR3FMQXdSVk1aTi9GNkR5dmF4WjlNL0dwbHJwVmJkem5DZXhFZVBwTFJSVmRveGtEOHlUK2xEVUFyNHZYM0pHdzBIK1RweG1Hb0JWRTMwRXpnbEJkSkp6enVXUlRDaGRLQllMTzFEWHR4eUQ0SkJNOWo0MlZ3bEl0V0dXTWhmT1NoNHl1U0hzVEtBOTlrayt0d0dINVpzcVhWc3llRVlOcldaV1JoSDVyMHpBSGkwS0VmNEd1c3pTbzNFbW1vdDRmTGpULzk1QzZEbHBQRFgyQkc1NExpOWJuNGRWcXEvVHFyaDJsbVp4YVUySkUyMlhHMzJqM0FLbkdJczZHY0cvTjBKN2xPc0RKWFdSVjdUMTZWTks4dUxOVUt0VEFTWmFyNmowSkVnTGJjdXlGZk9JTFNKL2tPRHdqUnhqcjBMVDExU3FGY0JhdlFyMGt5UmFOUi9ZZmMrYUhxM2ZIVkVZay9KWkpKY3J5Q2FmMEQ4eHE2end2eCt4M2Z3bTJ1bCtiRG5DMzl4MGRKbmNiMVo1ZGVHcm5SS0diRWdXU3FpeDFiVGlKdWNPRnNlanR1T3NFWGgyZzZ3ZHJzeGZrZG0xOFJzbVp1VXJaQzhrZjJxdFZEMkovdlh4TU5LWnVmdUo2cEM3N0d5TGxOMjhRU2tYcFdiWHNwNmdvRFJ0cHZsNWd2OGM3dGdhRkdNQkVnQWo2enZQRGNpa2pSM0ozSnIzTzJreGdneTBjVTlpUHJIU3c1RVAiLCJjc3JmVG9rZW4iOiIzNTc4OWI5NWVkODk3NWY4ZTg0MzJkN2NmNTgwMjhjYWUxOGM1MzI1LTE3MDIzMDkyNTc1NTgtYTFhZGI5YzJjNDY1ZDY2ODQzZjAxZDdiIn0sIm5iZiI6MTcwMjMwOTI1NywiaWF0IjoxNzAyMzA5MjU3fQ.HWNtstkK8DpJXVHMIOwMjh80YWwNGNLnIlDex28jtp0',
```

Comment:
(blocking) This will limit being able to run against other deployed instances. Rather than hard code specific environments let's pass it in an an environment variable. If the environment variable is not set and the base url is localhost/civiform:9000 fallback to that cookie, otherwise throw an error with instructions on how to get the cookie value.

Reply:
That's a great point.

Can you point me to an example of what I need to modify to get this to work?

Reply:
https://github.com/civiform/civiform/blob/67fae5075645e6a1c12c138068a621372d1dcdaf/browser-test/src/support/config.ts#L2

Reply:
and https://github.com/civiform/civiform/blob/67fae5075645e6a1c12c138068a621372d1dcdaf/browser-test/bin/wait_for_server_start_and_run_tests.sh#L8-L10